components/net/http_loader.rs

```diff
@@ -785,7 +786,7 @@ fn http_redirect_fetch(request: Rc<Request>,
     // TODO implement referrer policy
 
     // Step 13
-    main_fetch(request, cache, cors_flag, true, target, done_chan, context)
+    main_fetch(request, cache, cors_flag, recursive_flag, target, done_chan, context)
```

Comment:
We can follow https://github.com/whatwg/fetch/commit/869ec2c2224200988bbc604f7f5181b6bb9df2c3 now instead.